src/main/resources/org/jenkinsci/plugins/ownership/config/DisplayOptions/config.jelly

```diff
@@ -22,18 +22,18 @@
 * THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" 
-         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
+         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:o="/lib/ownership">
   <f:advanced title="${%Ownership Display Options}">
     <f:entry title="" description="">
-        <table width="100%">
+        <o:blockWrapper>
```

Comment:
What's the difference between this and using `<f:block>`

Reply:
block uses a `tr` not a `table`

seems to be that in a number of places `tr` and `td` were wrapped in tags but the top level table was not

Reply:
I see. Would it make sense to introduce something like this on Jenkins side?

Reply:
not sure, would have been great if it was there years ago, but may not be worth it now